src/components/ReportsPage/HeatmapPage/HeatmapPage.tsx

```diff
@@ -63,16 +67,25 @@ export const HeatmapPage = () => {
             onSelect={setSelectedScorecardId}
           />
         </Grid>
-        <Grid item style={{ marginTop: '20px' }}>
-          <GroupByDropdown groupBy={groupBy} setGroupBy={setGroupBy} />
+        <Grid container direction="row" style={{ marginTop: '20px' }}>
+          <Grid item>
+            <GroupByDropdown groupBy={groupBy} setGroupBy={setGroupBy} />
+          </Grid>
+          <Grid item>
+            <HeaderTypeDropdown
+              headerType={headerType}
+              setHeaderType={setHeaderType}
+            />
+          </Grid>
         </Grid>
         <Grid item lg={12}>
           {selectedScorecardId === undefined ? (
-            <AllScorecardsHeatmap groupBy={groupBy!!} />
+            <EmptyState title="Select a scorecard" missing="data" />
           ) : (
             <SingleScorecardHeatmap
               scorecardId={selectedScorecardId}
               groupBy={groupBy!!}
+              headerType={headerType!!}
```

Comment:
what does this double exclamation mark do?

Reply:
basically just says it'll never be undefined (which it wont)